hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -840,7 +820,7 @@ public Response pullRailsPortToDb(@Context HttpServletRequest request,
         GrailParams params = new GrailParams(reqParams);
         params.setUser(user);
         params.setWorkDir(workDir);
-        params.setOutput(layerName);
+        params.setInput1(layerName);
```

Comment:
I know there are some different uses of the grail params, i.e. overriding, where output value for one config is used as input for another, so just double check this is an intentional change.

Reply:
correct. at the start of setupRailsPull, which this params is sent to, i just save the name early on to 100% allow overwriting